src/AcceptanceTests/NServiceBus.DataBus.AzureBlobStorage.AcceptanceTests.csproj

```diff
@@ -14,9 +14,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.2.2" />
```

Comment:
I don't think this is the correct SDK. You need to use

https://www.nuget.org/packages/Azure.Storage.Blobs/